### ISSUE TITLE
`debug.log` still present with `-nodebug`, becomes very large

### ISSUE THREAD

**Issue by HarryR** [firoorg/firo#1011]
### Describe the issue

When performing an initial sync I noticed the `debug.log` file was filling up with messages very quickly, I don't want debug logging and none of the flags I tried via `--help` or `--help -help-debug` were able to stop the `debug.log` file from being created.

While syncing, for example:

 * chainstate: 40m
 * blocks: 12g
 * debug.log: 462mb

**I am concerned that potentially sensitive information may be logged in `debug.log`**, additionally it is causing a lot of additional I/O, and I have been unable to use program flags to turn it off.

There is no option to rotate the debug log, or limit it to a maximum size. And options like `-nodebug` don't seem to disable it.

### Can you reliably reproduce the issue?

Yes

#### If so, please list the steps to reproduce below:
1. `bin/firod -datadir=/chaindata -discover=0 -listen=0 -listenonion=0 -rest=1 -nodebug`
2. Verify that `/chaindata/debug.log` exists and is growing rapidly

### Expected behaviour
The `debug.log` file shouldn't exist

### Actual behaviour
The `debug.log` file contains many debug messages, while there is no status or progress information displayed on the console (program `stdout`), e.g.

```
2021-03-26 15:55:24 AcceptBlock nHeight=167741
2021-03-26 15:55:24 CheckBlock() nHeight=167741, blockHash= ed2a62c14eb9a5802f80a12b07d7dadc3de06cbf508c0594bb95a2bee7145aae, isVerifyDB = false
2021-03-26 15:55:24 CheckBlock() nHeight=2147483647, blockHash= 9de65ce64f49e0890bbd2db282898cfe74bf35d602775771a86c0a76c323aeb4, isVerifyDB = false
2021-03-26 15:55:24 CheckTransaction nHeight=167750, isVerifyDB=false, isCheckWallet=false, txHash=d907a67ee6515165b779f2e1acbefd24f88150091e78d091f8be31e1d533abd4
2021-03-26 15:55:24 CheckTransaction nHeight=167750, isVerifyDB=false, isCheckWallet=false, txHash=cae8b0fce703d32d8ddd613c98fe5defac9040bb7b019e541b37b1134718bacf
2021-03-26 15:55:24 CheckTransaction nHeight=167750, isVerifyDB=false, isCheckWallet=false, txHash=bffbb3e3756af4c8dceaa88ce01b7060681fd11900a976ab5bf07843765c9eeb
2021-03-26 15:55:24 CheckTransaction nHeight=167750, isVerifyDB=false, isCheckWallet=false, txHash=b7f7bee767afc4f96ae39625e887bfc5cfecb348d61d7600fd0eea0a9030ef4d
2021-03-26 15:55:24 AcceptBlock nHeight=167750
```

### What version of firo are you using?

0.14.5.3 from the github releases page

### Machine specs:
- OS: Linux
- CPU: Intel

**Comment by justanwar** [comment]
Have you tried starting with `-printtoconsole`?

**Comment by HarryR** [comment]
Thank you, `-printtoconsole` does work @justanwar - now debug.log isn't being filled up.

But now the firehose of verbose debug messages is being piped into console, I guess I can do `&> /dev/null`.

Is there any way to get a sane & sensible amount of information on the console or being logged, like zcash or monero that say 'Blockchain is 95% synced, estimated time 8.3hrs' and only spit out stuff when errors happen?

**Comment by justanwar** [comment]
Added a temporary flag fNoDebug. Bitcoin Core 0.17 introduces a redesign which have this logic fixed #1014
